### PR TITLE
PHP streaming retry and exception tests

### PR DESCRIPTION
**Pull request by michaelbausor**
This PR:
- Removes retry settings from PHP streaming methods. Retrying does not make sense in that context, and GAX prevents the use of retry settings for streaming methods in PHP
- Changes from INTERNAL errors to DATA_LOSS errors, because some methods are configured to retry on INTERNAL errors(!)
- Call getReceivedCalls in exception tests so that the stub will be exhausted

### CODE REVIEW

File: src/main/resources/com/google/api/codegen/clientconfig/json.snip [hunk 1]
```diff
@@ -88,12 +88,12 @@
                 retryCodesName = methodConfig.getRetryCodesConfigName, \
                 retryParamsName = methodConfig.getRetrySettingsConfigName
             "{@method.getSimpleName}": {
-              @if or(and(retryCodesName, retryParamsName), isBatching)
+              @if or(and(and(retryCodesName, retryParamsName), context.isRetryingSupported(method)), isBatching)
```
Comment (garrettjonesgoogle) [inline src/main/resources/com/google/api/codegen/clientconfig/json.snip]: Ack I didn't realize that clientconfig generation wasn't on MVVM. Since this is totally outside the scope of this PR, can you make sure an issue is filed if one isn't yet?

File: src/main/java/com/google/api/codegen/clientconfig/php/PhpClientConfigGapicContext.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+public class PhpClientConfigGapicContext extends ClientConfigGapicContext {
+
+  /** Constructs the client config codegen context. */
+  public PhpClientConfigGapicContext(Model model, GapicProductConfig config) {
+    super(model, config);
+  }
+
+  @Override
+  public boolean isRetryingSupported(Method method) {
+    return isSupported(method) && !method.getRequestStreaming() && !method.getResponseStreaming();
```
Comment (garrettjonesgoogle) [inline src/main/java/com/google/api/codegen/clientconfig/php/PhpClientConfigGapicContext.java]: This seems applicable to all languages - maybe get all languages to sign off?

File: src/main/resources/com/google/api/codegen/clientconfig/json.snip (hunk 1 above)
Comment (michaelbausor) [inline src/main/resources/com/google/api/codegen/clientconfig/json.snip]: https://github.com/googleapis/toolkit/issues/1307

File: src/main/java/com/google/api/codegen/clientconfig/php/PhpClientConfigGapicContext.java (hunk 2 above)
Comment (michaelbausor) [inline src/main/java/com/google/api/codegen/clientconfig/php/PhpClientConfigGapicContext.java]: I don't think that it is. Other languages use retries here, to retry stream creation. But that doesn't make sense in PHP because we do not receive a status immediately.

**Comment by garrettjonesgoogle** [comment]
LGTM
